jwt.go

```diff
@@ -153,9 +153,11 @@ func ParsePrivateKey(data []byte) (interface{}, error) {
 
 // GetTokenFromRequest takes the first Authorization header or `token` GET pararm , then
 // extract the token prefix and json web token
-func GetTokenFromRequest(r *http.Request) (prefix string, token string, err error) {
-
-	tokenList, ok := r.Header[AuthorizationHeader]
+func GetTokenFromRequest(r *http.Request, header string) (prefix string, token string, err error) {
+	if header == "" {
+		header = AuthorizationHeader
+	}
+	tokenList, ok := r.Header[header]
```

Comment:
We should probably use `https://golang.org/pkg/net/http/#CanonicalHeaderKey` here?